.github/workflows/run_tests.yml

```diff
@@ -17,16 +17,31 @@ jobs:
     - name: checkout git repo
       uses: actions/checkout@v3
 
-    - name: pull sdk image and test
-      shell: bash
-      env:
-        KBASE_TEST_TOKEN: ${{ secrets.KBASE_TEST_TOKEN }}
-        GITHUB_ACTIONS_CI: 1
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+
+    - name: install prerequisites
+      shell: bash -l {0}
+      run: |
+        cat requirements.txt | sed -e '/^\s*#.*$/d' -e '/^\s*$/d' | xargs -n 1 pip install
+        cat requirements-test.txt | sed -e '/^\s*#.*$/d' -e '/^\s*$/d' | xargs -n 1 pip install
```

Comment:
split up the requirements into three files (there were two) -- requirements, requirements-test (for running/testing), and requirements-dev (for development).

Reply:
(See readme file for details)

Reply:
Why not just `pip install -r requirements.txt` here?

Reply:
Ah, just saw the README comment. That's frustrating. 👍